ClientSide(FrontEnd)/clay_/src/components/login.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
use one line for importing react and it hooks like below
import React, { useState } from 'react';

Reply:
i added both in one line